server.go

```diff
@@ -447,6 +453,42 @@ func newServer(listenAddrs []string, chanDB *channeldb.DB, cc *chainControl,
 			_, err := cc.wallet.GetPrivKey(addr)
 			return err == nil
 		},
+		ContractBreach: func(chanPoint wire.OutPoint,
+			breachRet *lnwallet.BreachRetribution) error {
+			event := &ContractBreachEvent{
+				ChanPoint:         chanPoint,
+				ProcessACK:        make(chan error, 1),
+				BreachRetribution: breachRet,
+			}
+
+			// Send the contract breach event to the breachArbiter.
+			select {
+			case contractBreaches <- event:
+			case <-s.quit:
+				return fmt.Errorf("exiting")
+			}
+
+			// Wait for the breachArbiter to ACK the event.
+			select {
+			case err, ok := <-event.ProcessACK:
+				if !ok {
```

Comment:
When does `ProcessAck` get closed? I was unable to find the call

Reply:
It doesn't, did the err check just to make sure we only return a non-nil error in the case we actually send a non-error. Maybe safe to remove this check..?